### DISCUSSION TITLE
What is the easiest way for object detection during live streams?

### DISCUSSION THREAD

**Discussion by umtcylmz** [ant-media/Ant-Media-Server#5520] [FAQs] [answered]
I want to detect cars, bicycles, etc. during my live streams. Is there any plugin for that on top of Ant Media Server? If so, how can I ask for a demo?

**Comment by Mohit-3196** [comment, marked as answer]
Ant Media Server offers a powerful solution for real-time object detection during live streams through the **[Ant Media Server Marketplace](https://antmedia.io/marketplace/)** called **[DirectAI](https://antmedia.io/computer-vision/)**. This remarkable tool is designed to make object detection seamless and accessible, allowing you to identify and track various objects like cars, bicycles, and more during your live streams.

**[DirectAI](https://antmedia.io/computer-vision/)**: **DirectAI** takes the complexity out of the equation by providing an intuitive and efficient way to incorporate computer vision into your streaming workflows. What's truly remarkable is that you can achieve all of this without the need for extensive training data or coding skills. DirectAI leverages Vision Language Models (VLMs) that have already been trained on an immense volume of image-caption pairs. This means you can create and deploy computer vision models in mere seconds without the traditional hurdles.

**How to Get Started:**
Getting started with DirectAI and enhancing your live streams with object detection is a straightforward process:

- Explore the Marketplace: Head over to the [Ant Media Marketplace](https://antmedia.io/marketplace/) and navigate to the [DirectAI page](https://antmedia.io/computer-vision/).

- Learn More: Dive into the detailed information provided on the page to understand how DirectAI can empower your streaming experience.

- Request a Demo: If you're intrigued and want to see DirectAI in action, you can request a demo directly from the website.

Thank you.

**Comment by lastpeony** [comment]
tensorflow plugin is also very useful
https://github.com/ant-media/Plugins/tree/master/TensorflowPlugin
